docs/sorting-algorithms/Counting-sort.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: Counting Sort
+slug: /sorting-algorithms/Counting-sort
+---
+
+## Introduction
+
+Counting Sort is a sorting algorithm that sorts the element of array by counting the number of occurence of each distinct elements. In this we have store the occurence of element in count array and then modify the count array such that every index store the sum of previous counts. And this modified count array shows that how many elements are smaller than that element. Then make the final array that contains sorted elements.Make sure that elements are in small range.
+
+## Explanation
+First store the occurence of each elements in count array. Then modify the count array by storing the sum of counts of previous index and this modified array tell us how many elements are smaller than that elements. Make an output array which contained sorted elements. Output array is build with the help of given(input) array and modified count array by traversing the input array from right to left(last to first).
+
+### Example 1
+
+<img src="https://i.imgur.com/2TkzRQz.png" width="500" height="350"/>
+
+```
+
+ Array arr[4]:        
+ | 5 | 1 | 4 | 1 | 3 | 1 | 2 | 5 |                               
+ Store the occurence in count array
+ | 0 | 3 | 1 | 1 | 1 | 2 | 0 | 0 | 0 | 0 |
+ Modify the count array
+ | 0 | 3 | 4 | 5 | 6 | 8 | 8 | 8 | 8 | 8 |
+then make an output array from given array and modified count array
+| 1 | 1 | 1 | 2 | 3 | 4 | 5 | 5 |
+  Sorted!! 
+
```

Comment:
Can you follow a slightly more standard example? Take a look at the existing examples and see if you can format yours according to them.